frontend/android/src/main/java/io/github/droidkaigi/confsched2019/App.kt

```diff
@@ -77,6 +88,15 @@ open class App : DaggerApplication() {
         EmojiCompat.init(config)
     }
 
+    private fun setupSystemActionCreator() {
+        ProcessLifecycleOwner.get().lifecycle.addObserver(this)
```

Comment:
Can you move this to onCreate? Because it can be used by other action creator.

Reply:
OK, I fix it :)